pkg/kubectl/cmd/cmd.go

```diff
@@ -348,6 +357,7 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 			},
 		},
 	}
+	groups.PruneCommands()
```

Comment:
When a command disables itself by returning `nil` it's as if the command doesn't exist, which is intentional. My goal is that alpha features don't influence the behavior of `kubectl` in any way unless they're enabled.